modules/ROOT/partials/network-status.adoc

```diff
@@ -0,0 +1,78 @@
+// Type: procedure
+[id='network-service-{context}']
+= Exploring a {service-network}
+
+{skupper-name} includes a command to allow you report all the sites and the services available on a {service-network}.
+
+.Prerequisites
+
+* A {service-network} with more than one site
+
+.Procedure
+
+. Set your Kubernetes context to a namespace on the {service-network}.
+
+. Use the following command to report the status of the {service-network}:
++
+--
+
+[source,bash]
+----
+$ skupper network status
+----
+
+For example, the following shows the output for the {service-network} created from the {OpenShiftTutoriallink} tutorial from the `west` namespace:
+
+[source]
+----
+Sites:
+├─ [local] 4dba248 - west  <1>
+│  URL: 10.96.146.236 <2>
+│  name: west <3>
+│  namespace: west
+│  version: 0.8.6 <4>
+│  ╰─ Services:
+│     ╰─ name: hello-world-backend <5>
+│        address: hello-world-backend: 8080 <6>
+│        protocol: tcp <7>
+╰─ [remote] bca99d1 - east <8> 
+   URL: 
```

Comment:
Is it worth noting a blank URL occurs when "--ingress none" or iiuc when in ''--router-mode" edge"?

Reply:
added note at end to explain this with '--ingress none'.. we don't mention edge router elsewhere, so avoiding that